signicat/Issues/kvkTradeRegister/description.xml

```diff
@@ -0,0 +1,463 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Official of a legal entity</en>
+		<nl>Functionaris voor een rechtspersoon</nl>
+	</Name>
+	<ShortName>
+		<en>Official</en>
+		<nl>Functionaris</nl>
+	</ShortName>
+	<SchemeManager>pbdf</SchemeManager>
+	<IssuerID>signicat</IssuerID>
+	<CredentialID>kvkTradeRegister</CredentialID>
+	<Description>
+		<en>The personal information of an official of a legal entity</en>
+		<nl>De persoonlijke informatie van een functionaris voor een rechtspersoon</nl>
+	</Description>
+	<IssueURL>
+		<en>https://organisatiegegevens.signicat.nl</en>
+		<nl>https://organisatiegegevens.signicat.nl</nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="kvkNumber">
+			<Name>
+				<en>KVK number</en>
+				<nl>KVK nummer</nl>
+			</Name>
+			<Description>
+				<en>The KVK number of the organisation</en>
+				<nl>Het KVK nummer van de organisatie</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="rsin" optional="true">
+			<Name>
+				<en>RSIN number</en>
+				<nl>RSIN nummer</nl>
+			</Name>
+			<Description>
+				<en>Legal Entity or Partnership Identification Number</en>
+				<nl>Rechtspersoon of Samenwerkingsverband Identificatie Nummer</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="name">
+			<Name>
+				<en>Name of the organisation</en>
+				<nl>Naam van de organisatie</nl>
+			</Name>
+			<Description>
+				<en>The name of the organisation</en>
+				<nl>De naam van de organisatie</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="tradeNames" optional="true">
+			<Name>
+				<en>Trade names</en>
+				<nl>Handelsnamen</nl>
+			</Name>
+			<Description>
+				<en>The trade names under which the company trades</en>
+				<nl>De handelsnamen waar de onderneming onder handelt</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="typeOwner">
+			<Name>
+				<en>Type of owner</en>
+				<nl>Type eigenaar</nl>
+			</Name>
+			<Description>
+				<en>The type of owner of the organisation</en>
+				<nl>De type eigenaar van de maatschappelijke activiteit</nl>
```

Comment:
'Het' type

Some 'description' fields start with an article, some not. To make this more consistent maybe remove the first article' in all description-fields